gotocompany/compass/v1beta1/service.proto

```diff
@@ -206,14 +206,30 @@ service CompassService {
 
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
       summary: "Search for an asset";
-      description: "API for querying documents. 'text' is fuzzy matched against all the available datasets, and matched results are returned. You can specify additional match criteria using 'filter[.*]' query parameters. You can specify each filter multiple times to specify a set of values for those filters. For instance, to specify two landscape 'vn' and 'th', the query could be `/search/?text=<text>&filter[environment]=integration&filter[landscape]=vn&filter[landscape]=th`. As an alternative, this API also supports fuzzy filter match with 'query' query params. For instance, searching assets that has 'bigqu' term in its description `/search/?text=<text>&query[description]=bigqu`";
+      description: "API for querying documents. 'text' is fuzzy matched against all the available datasets, and matched results are returned. You can specify additional match criteria using 'filter[.*]' query parameters. You can specify each filter multiple times to specify a set of values for those filters. For instance, to specify two landscape 'vn' and 'th', the query could be `/search/?text=<text>&filter[environment]=integration&filter[landscape]=vn,th`. As an alternative, this API also supports fuzzy filter match with 'query' query params. For instance, searching assets that has 'bigqu' term in its description `/search/?text=<text>&query[description]=bigqu`";
       tags: [
         "Search",
         "Asset"
       ];
     };
   }
 
+  // Domain: Asset
+  rpc GroupAssets(GroupAssetsRequest) returns (GroupAssetsResponse) {
+    option (google.api.http) = {
+      get: "/v1beta1/groupassets"
+    };
+
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Group api for assets";
+      description: "API for grouping documents. 'groupby' the field provided here is used for grouping. You can provide multiple fields as well. If it is a nested fields, you can provide the path to the field. You can specify additional criteria using 'filter[.*]' to only group those documents. You can specify each filter multiple times to specify a set of values for those filters. For instance, to specify two landscape 'vn' and 'th', the query could be `/groupassets/?groupby=<groupby>&filter[environment]=integration&filter[landscape]=vn&filter[landscape]=th`. 'include_fields' is used to send only these fields in response. this helps in restricting the response. Eg:/groupassets/?groupby=field1,field2&filter[environment]&include_fields=data.attributes.dataset,data.attributes.project Fields in group by will always be part of response, no need to add them in include-fields.";
```

Comment:
This description still has an incorrect example.

Reply:
Fixed